app/Http/Requests/UserStoreRequest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\BaseRequest as BaseRequest;
+
+class UserStoreRequest extends BaseRequest
+{
+    public const NAME = 'name';
+    public const EMAIL = 'email';
+    public const PASSWORD = 'password';
+    public const ROLE = 'role';
+    public const BALANCE = 'balance';
+
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            self::NAME => [
+                'required',
+                'string',
+                'max:255'
+            ],
+            self::EMAIL => ['required',
+                'email',
+                'unique:users',
+                'email'
+            ],
+            self::PASSWORD => ['required', '
+            string',
+                'min:6'
+            ],
```

Comment:
```suggestion
            self::PASSWORD => [
                'required',
                'string',
                'min:6',
            ],
```

Нужно написать в таком формате